README.md

```diff
@@ -108,3 +122,58 @@ git push --tags
 Once you've pushed the tags, the new release will appear in the project's
 ["Releases" tab in GitHub](https://github.com/azavea/idb-osm-extraction-tool/releases).
 There you can edit the tagged release to add release notes.
+
+### Feature configuration
+OSM tags are grouped into high level features in [the config file](src/js/src/featureConfig.js).
+The configuration can take all values of a tag, only certain keys of a
+tag, and multiple tags. A comprehensive example of a feature configuration is:
+
+```javascript
+{
+        label: 'Shop/Business',
+        entities: [
+            { tag: 'shop' },
+            { tag: 'craft' },
+            { tag: 'office' },
+            {
+                tag: 'building',
+                values: ['office'],
+            },
+            {
+                tag: 'amenity',
+                values: ['restaurant', 'cafe', 'internet_cafe', 'bar', 'biergarten', 'fast_food', 'marketplace', 'fuel'],
+            },
+        ],
+    },
+```
+
+The OSM tags will be converted to an [Overpass API query](https://wiki.openstreetmap.org/wiki/Overpass_API/Language_Guide) and fetched from that service.
+
+### Code of Conduct 
+---
+
+### Authors
+---
+
+* [@kellyi](https://github.com/kellyi)
+* [@mmcfarland](https://github.com/mmcfarland)
+* [@rbreslow](https://github.com/rbreslow)
+* [@eneedham](https://github.com/eneedham)
+* [@rajadain](https://github.com/rajadain)
+* [@alexlash](https://github.com/alexelash)
+
+For full list see [Contributors](https://github.com/azavea/idb-osm-extraction-tool/graphs/contributors)
+
+### License 
+---
+
+Licensed under [AM-331-A3 Licencia de Software](LICENSE.md)
+
+### Limitation of responsibilities
+---
+
+The IDB is not responsible, under any circumstance, for damage or compensation, moral or patrimonial; direct or indirect; accessory or special; or by way of consequence, foreseen or unforeseen, that could arise:
+
+I. Under any concept of intellectual property, negligence or detriment of another part theory; I
+
+ii. Following the use of the Digital Tool, including, but not limited to defects in the Digital Tool, or the loss or inaccuracy of data of any kind. The foregoing includes expenses or damages associated with communication failures and / or malfunctions of computers, linked to the use of the Digital Tool.
```

Comment:
Nitpick but it looks like the numbering scheme here is

```
I.
ii.
```

Just wanted to make sure that is intentional.

Reply:
Hmm, good catch. I copied that verbatim from the template, https://github.com/azavea/idb-omk-pilot/blob/0cc6c29b27e124ad709f6331fa61924e606ad8ff/README-EN.md

Looks pretty unusual to me, but I'm hesitant to change something that looks like it was crafted by lawyers. I'll point it out to IDB folks and then maybe fix it up when we get remaining content.